app/services/back_end_service.rb

```diff
@@ -34,6 +34,15 @@ def gyms_near_user(zip_code)
       parse_json(response.body)
     end
 
+    def get_one_gym(yelp_gym_id)
+      response = db_conn.get("/api/v1/gym_search/#{yelp_gym_id}")
+      parse_json(response.body)
+    end
+
+    def post_gym_membership(gym_membership_params)
+      db_conn.post("/api/v1/users/#{gym_membership_params[:user_id]}/gym_memberships", gym_membership_params.to_json,'Content-Type' => 'application/json')
```

Comment:
I would suggest new lines, similar to the `db_conn.post()` method (for readability)

Not important at this point - but we could also abstract the headers to a separate method (`'Content-Type' => 'application/json'`) so we can just call `BackEndService.headers` (or something) to shorten these methods further

Reply:
Resolved long line. Later refactor for second comment.